mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -2452,6 +2536,16 @@ static void insertValidationCalls(const GenParams &genParams, OpBuilder &b,
           valVars.resize(valVars.size() - 1);
         }
 
+        // Make the kernel and its wrapper function, and call it.
```

Comment:
I had moved this when messing with createGPUWrapper(), to have the parameter list trimmed when I touched it.  Now I don't mess with createGPUWrapper() and could undo this specific change.  It's only a block swap;  the only code change is removing the unused variable newConfig.

Reply:
I think `newConfig` was needed to capture feature/type changes arising from creating the validation kernel. I'm a bit suspicious that it's going away